bundles/binding/org.openhab.binding.xbmc/src/main/java/org/openhab/binding/xbmc/rpc/calls/PlayerGetItem.java

```diff
@@ -80,23 +80,29 @@ public String getPropertyValue(String property){
 		Object value = item.get(paramProperty);
 
 		if (value instanceof List<?>) {
-			// some properties come back as a list with an indexer
-			String paramPropertyIndex = getPropertyValue(paramProperty + "id");
-			if (StringUtils.isEmpty(paramPropertyIndex))
-				return null;
+			List<?> values = (List<?>)value;
 			
-			// attempt to parse the property index
-			int propertyIndex;
-			try {
-				propertyIndex = Integer.parseInt(paramPropertyIndex);
-			} catch (NumberFormatException e) {
+			// check if list contains any values
+			if (values.size() == 0)
 				return null;
-			}
 			
-			// check if the index is valid
-			List<?> values = (List<?>)value;
-			if (propertyIndex < 0 || propertyIndex > values.size())
-				return null;
+			// some properties come back as a list with an indexer
+			String paramPropertyIndex = getPropertyValue(paramProperty + "id");
+			// other properties come back as a list without an indexer,
+			// e.g. artist, for these we return the first in the list
+			int propertyIndex = 0;
+			if (!StringUtils.isEmpty(paramPropertyIndex)) {
```

Comment:
It looks fine Marcel - the only thing I would do differently is have an explicit check for a missing `paramPropertyIndex` and return the first element in the array - just so it is obvious to anyone reading the code that in this case if there is a missing indexer we just return the first item. Minor refactoring tho, it looks fine to me otherwise.